wgpu-types/src/lib.rs

```diff
@@ -828,6 +945,44 @@ pub struct DepthStencilStateDescriptor {
 }
 
 impl DepthStencilStateDescriptor {
+    pub fn new(format: TextureFormat) -> Self {
+        Self {
+            format,
+            depth_write_enabled: false,
+            depth_compare: CompareFunction::Undefined,
+            stencil_front: StencilStateFaceDescriptor::default(),
+            stencil_back: StencilStateFaceDescriptor::default(),
+            stencil_read_mask: 0,
+            stencil_write_mask: 0,
+        }
+    }
+
+    pub fn depth_write(&mut self) -> &mut Self {
+        self.depth_write_enabled = true;
+        self
+    }
+
+    pub fn depth_compare(&mut self, compare: CompareFunction) -> &mut Self {
+        self.depth_compare = compare;
+        self
+    }
```

Comment:
Should `depth_write` and `depth_compare` be combined?

Reply:
nope